.gitignore

```diff
@@ -64,3 +64,6 @@ www/test_out
 # precommit temporary directories created by ./hack/verify-gendocs.sh and ./hack/lib/util.sh
 _tmp/
 doc_tmp/
+
+# contrib ansible yml example
+/contrib/ansible/group_vars/all.yml
```

Comment:
this should go in either `contrib/ansible/.gitignore` or `contrib/ansible/group_vars/.gitignore`.   I guess I'd like to see both all.yml and inventory in the former...